packages/attestation-service/config/.env.development

```diff
@@ -45,6 +45,7 @@ TWILIO_UNSUPPORTED_REGIONS=CU,SY,KP,IR,SD
 
 # MessageBird parameters (fill in values for 'x')
 MESSAGEBIRD_API_KEY=x
+MESSAGEBIRD_UNSUPPORTED_REGIONS=CU,SY,KP,IR,SD
```

Comment:
do we want to add messagebird to `SMS_PROVIDERS` or wait until we feel more confident about it as an option?

Reply:
mm this is a release process edge case. Because the docs reference this config file from master branch, we don't want to include anything that is not backwards compatible with the current released version. Because MessageBird is not yet in a released version, adding it to `SMS_PROVIDERS` would break things.  We should add it, but only just as we release the next version I guess.